test/kitchen/drivers/azure-driver.yml

```diff
@@ -46,9 +57,13 @@ driver_config:
   custom_data: |
     #cloud-config
     runcmd:
-      - mkdir /mnt/kitchen
-      - chmod a+rwx /mnt/kitchen
-      - ln -s /mnt/kitchen /tmp/kitchen
+      - mkdir <%= mount_dir %>
+      - chmod -R a+rwx <%= mount_dir %>
+      - ln -s <%= mount_dir %> <%= target_mount_dir %>
+      <% if chang_mount_dir %>
+      - mkdir <%= mount_dir %>/kitchen
+      - chown -R <%= ENV['VM_USERNAME'] ||= "datadog" %>:<%= ENV['VM_USERNAME'] ||= "datadog" %> <%= target_mount_dir %>/kitchen
```

Comment:
I don't think we use the `datadog` user on EC2 Linux instances.

Reply:
indeed, but the change is in azure driver

Reply:
Oops! My bad.